constraints/binarity.js

```diff
@@ -79,35 +79,69 @@ function binMaxBranchesGradient(s, ptree, cat){
 
 /*TRUCKENBRODT-STYLE BINARITY*/
 
-//Parent-category-neutral version of:
-//Sandalo & Truckenbrodt 2002: "Max-Bin: P-phrases consist of maximally two prosodic words"
-//Assigns a violation for every node in ptree that dominates more than two prosodic words.
```

Comment:
We should keep the part of the old comment about Sandalo & Truckenbrodt 2002 back in; the source of the original constraint proposal is useful information.

Reply:
comment restored in latest commit starting on line 87